### ISSUE TITLE
[BUG-FIXING BOUNTY - 20 USD] Weapon jamming bug

### ISSUE THREAD

**Issue by evilew** [KabKebab/GS13#188]
**tl;dr 20 USD via PayPal for fixing a bug that causes most weapons to "jam". Needs to be tested, documented and explained to receive payment.**

**Details:**
The bug results in player characters being unable to fire a weapon completely for a long period of time (usually anywhere from couple seconds to several minutes), usually after taking a single or two shots. 

This happens most frequently on energy or self-replenishing weapons, such as laser rifles, or proto-kinetic accelerators. Ballistic weapons seem to suffer from the bug far less frequently though there is one report of it happening to the stechkin pistol (requires further testing). The bug does not affect plasma cutters at all.

Players have reported that sometimes dropping or hiding the weapon in the backpack will reset it and it will be fireable again. The bug seems to occur randomly, sometimes not happening several rounds and sometimes appearing for several shifts in a row.

**Possible causes:**
- timers
- there was a brief rumor that round ID correlates to this, but there is no evidence that'd support this claim

**More information will be added whenever it's available.**
